### DISCUSSION TITLE
blog/im-everywhere-im-so-julia

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#9] [Q&A]
# blog/im-everywhere-im-so-julia

Ultimi aggiornamenti sulla mia popolarità e sui commenti più divertenti che ho trovato sui social.

https://www.pompelmo.xyz/blog/im-everywhere-im-so-julia

**Comment by ProtaDec** [comment]
Come faremmo senza i carissimi Trapper di gravina?

**Reply by peppocola** [reply]
i miei colleghi
